packages/react-native-bridge/index.js

```diff
@@ -489,7 +494,12 @@ export function showAndroidSoftKeyboard() {
 		return;
 	}
 
-	RNReactNativeGutenbergBridge.showAndroidSoftKeyboard();
+	const hasFocusedTextInput =
+		RCTAztecView.InputState.getCurrentFocusedElement() !== null;
```

Comment:
Nitpick: [`isFocused` function](https://github.com/WordPress/gutenberg/blob/cc77161ebfc89667fd66848a670ca6fceebbee3b/packages/react-native-aztec/src/AztecInputState.js#L95-L97) already provides this value, hence we could simply call:
```suggestion
	const hasFocusedTextInput = RCTAztecView.InputState.isFocused();
```

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/57279/commits/00597901147cff714288e3016233fa35097e6ece thanks for the suggestion!